.github/workflows/ci-nimbus.yml

```diff
@@ -14,28 +19,24 @@ jobs:
         target:
           - os: linux
             cpu: amd64
-          # - os: linux
-          #   cpu: i386
-          - os: macos
-            cpu: amd64
-          - os: windows
-            cpu: amd64
-          #- os: windows
-            #cpu: i386
+          # - os: macos
+          #   cpu: amd64
+          # - os: windows
+          #   cpu: amd64
         branch: [version-1-6]
         include:
           - target:
               os: linux
             builder: ubuntu-20.04
             shell: bash
-          - target:
-              os: macos
-            builder: macos-12
-            shell: bash
-          - target:
-              os: windows
-            builder: windows-2019
-            shell: msys2 {0}
+          # - target:
```

Comment:
why get rid of the targets&CPUs?

Reply:
I figured it didn't make sense to run the targets again for all the arch's / cpu's. This is mainly to help ensure nimbus build setup doesn't decay. 

Though we could run both nimble & nimbus for each arch.

Reply:
> I figured it didn't make sense to run the targets again for all the arch's / cpu's. This is mainly to help ensure nimbus build setup doesn't decay.
> 
> Though we could run both nimble & nimbus for each arch.

Ah, I see. I think at this point, we might be safe to remove NBS altogether? I know we discussed having both, but it looks like it isn't adding much right now?

Reply:
I created an issue to remove it #71